tests/bson/bson-iterator-iterator_error-001.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+MongoDB\BSON\Iterator cannot be rewound
```

Comment:
Per an earlier comment, I see no reason we can't allow rewinding these iterators.